app/controllers/api/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Brian Ploetz (bploetz@gmail.com)
```

Comment:
Well, that's a easy case. That code was made by nov and there is only novs code in the file, so we have to add a license header here.